app/grandchallenge/hanging_protocols/forms.py

```diff
@@ -183,20 +183,23 @@ def __init__(self, *args, **kwargs):
         )
 
     def _get_interface_lists(self):
+        sorted_interfaces = sorted(
+            list(self.instance.linked_component_interfaces), key=lambda x: x.pk

```

Comment:
I remember that Chris recently worked on this. Does it make sense to sort by the pk? I thought this was sorted by slug?

Reply:
Sorting by slug is also fine. This is only used for generating the view_content examples, so doesn't really matter as long as they are sorted somehow. I will update it to sorting by slug.